build.gradle

```diff
@@ -86,8 +84,8 @@ dependencies {
     testImplementation 'org.slf4j:slf4j-api:1.7.30'
     testImplementation 'com.github.tomakehurst:wiremock-standalone:2.27.2'
     testImplementation 'io.jsonwebtoken:jjwt:0.9.1'
-    testImplementation 'io.swagger.parser.v3:swagger-parser-v2-converter:2.0.22'
     testImplementation 'org.glassfish:javax.el:3.0.0'
+    testImplementation 'io.swagger.parser.v3:swagger-parser:2.0.20'
```

Comment:
required to downgrade due to json deserialization bug introduced here https://github.com/swagger-api/swagger-parser/pull/1400 and discussed here https://github.com/tipsy/javalin/issues/1079 and here https://github.com/swagger-api/swagger-core/pull/3637